models/Asset.php

```diff
@@ -1562,28 +1562,20 @@ public function getMetadata($name = null, $language = null, $strictMatch = false
             $data = null;
             foreach ($this->metadata as $md) {
                 if ($md['name'] == $name) {
-                    if ($language == $md['language']) {
-                        if ($raw) {
-                            return $md;
-                        }
-
-                        return $convert($md);
-                    }
                     if (empty($md['language']) && !$strictMatch) {
                         if ($raw) {
                             return $md;
```

Comment:
this part could be removed because it's converted below when $data is set at the end of the loop, but it looks like a bug unrelated to the change on this PR